packages/backend/src/controllers/house/house.controller.ts

```diff
@@ -68,22 +68,25 @@ export class HouseController {
     description: 'house retrieved successfully',
     type: HouseResponseDto,
   })
+  @ApiNotFoundResponse({ description: 'user is not in a house' })
   async getHouse(
     @User() user: DecodedIdToken,
   ): Promise<HouseResponseDto | null> {
     const userDoc = await this.userStoreService.findOneByFirebaseId(user.uid);
-    const house = await this.houseStoreService.findOne(userDoc.house);
-    if (house) {
-      return {
-        code: house.code,
-        address: house.address,
-        email: house.email,
-        owner: user.uid,
-        name: house.name,
-      };
-    } else {
-      return null;
+    if (userDoc.house != undefined) {
```

Comment:
Should we be using the strict equality operator?

Reply:
![image](https://user-images.githubusercontent.com/7777317/158038185-72acbc94-2d6d-4692-9c23-b2224ca4798c.png)
I used the equality operator because it can catch a few more edge cases